src/core/SampleBuffer.cpp

```diff
@@ -247,7 +247,15 @@ void SampleBuffer::update(bool keepSettings)
 	const int fileSizeMax = 300; // MB
 	const int sampleLengthMax = 90; // Minutes
 
-	bool fileLoadError = false;
+	enum FileLoadError
```

Comment:
This would be better as an `enum class`. (https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class)